.github/workflows/reusable-e2e.yml

```diff
@@ -384,6 +384,10 @@ jobs:
         if: steps.restore-package-cache.outputs.cache-hit != 'true'
         run: yarn react-core build
 
+      - name: Build react-core-notifications package
+        if: steps.restore-package-cache.outputs.cache-hit != 'true'
+        run: yarn react-core-notifications build
```

Comment:
Please add `react-core-notifications` package to the list of packages to check for changes and run RN tests (line 326 above, `has-rn-changed-packages` step)

Reply:
Added in latest commit.